airbyte-integrations/connectors/source-github-singer/setup.py

```diff
@@ -33,7 +33,13 @@
     package_data={"": ["*.json"]},
     # two sets of dependencies: 1) for main 2) for standard test deps. 2 does not have all of the dependencies of 1, which is we cannot use install_requires.
     extras_require={
-        "main": ["tap-github==1.9.0", "requests", "airbyte-protocol", "base-singer", "base-python"],
+        "main": [
+            "tap-github @ https://github.com/vitaliizazmic/tap-github/tarball/master#egg=tap-github-1.9.1",
```

Comment:
is it possible to use this fork instead: https://github.com/airbytehq/tap-github ? it is identical to yours, but having it under the Airbyte organization/brand is good for external visibility.

Reply:
Yes, I've replaced it.